tmva/tmva/inc/TMVA/Event.h

```diff
@@ -71,6 +71,9 @@ namespace TMVA {
 
       ~Event();
 
+      // operators
+      Event& operator=( const Event& ) = default;
```

Comment:
I don't understand why the copy ctor has the section
```
   if (event.fDynamic){
```
while `operator=` does not need it. Could you explain in the commit log, please?

Reply:
I put a comment in the commit message and the header that we do not want to change the behaviour since it's a public interface and not used internally.